TSP/BLL/Services/EmployeeService.cs

```diff
@@ -44,6 +44,10 @@ public async Task<Employee> GetEmployeeByIdAsync(int id)
         {
             var employee = await _repository.GetByIdAsync(id);
             var mappedEmployee = _mapper.Map<Employee>(employee);
+            if(employee is null)
```

Comment:
Should be checked after getting the employee

Reply:
fix